includes/wpmenucart-woocommerce.php

```diff
@@ -11,7 +11,7 @@ public function menu_item() {
 			$this->maybe_load_cart(); // make sure cart is loaded! https://wordpress.org/support/topic/activation-breaks-customise?replies=10#post-7908988
 
 			$menu_item = array(
-				'cart_url'            => wc_get_cart_url(),
+				'cart_url'            => apply_filters( 'woocommerce_get_cart_url', wc_get_page_permalink( 'cart' ) ),
```

Comment:
I think we could improve this by having our own filter on top of the Woo default:

```php
public function get_cart_url() {
	$wc_cart_url = apply_filters( 'woocommerce_get_cart_url', wc_get_page_permalink( 'cart' ) );
	return apply_filters( 'wpmenucart_cart_url', $wc_cart_url );
}
```

Reply:
Maybe we could also pass the class object.

Reply:
@MohamadNateqi :arrow_up:

Reply:
Done.